roles/provision-buildfarm-apb/files/install-jenkins-plugins.sh

```diff
@@ -27,10 +27,10 @@ oc cp -n ${namespace} /tmp/jenkins-plugins/android-signing.jpi ${jenkins_pod}:/v
 
 echo "Scaling down Jenkins"
 
-oc scale dc jenkins --replicas=0 -n ${namespace}
-wait_for_scale "jenkins" 0 ${namespace}
+oc scale dc/"{{ ag_jenkins.deployment.name }}" --replicas=0 -n ${namespace}
+wait_for_scale "{{ ag_jenkins.deployment.name }}" 0 ${namespace}
 
 echo "Scaling up Jenkins"
 
-oc scale dc jenkins --replicas=1 -n ${namespace}
-wait_for_scale "jenkins" 1 ${namespace}
+oc scale dc/"{{ ag_jenkins.deployment.name }}" --replicas=1 -n ${namespace}
+wait_for_scale "{{ ag_jenkins.deployment.name }}" 1 ${namespace}
```

Comment:
@vchepeli The changes to this script appear to have broken the installation of the plugins. I noticed before these changes were made that the plugins install could fail if the Jenkins instance takes a long time to scale up initially e.g if it had to pull a new image, but now it never seems to work and will fall straight through this script onto the SDK installation.

@vchepeli Any ideas?

Reply:
Ahh yep, that template properties are used should should not be there, passing it as second argument to script can fix problem